termCombinationLib.py

```diff
@@ -216,6 +218,26 @@ def removeTermsSmallerThanMinTermSize(tbsGsIDs, geneSetsDict, minTermSize):
 			tbsGsIDs.remove(idToRemove)
 	return tbsGsIDs
 
+
+def removeTermsLargerThanMaxTermSize(tbsGsIDs, geneSetsDict, maxTermSize):
+	"""
+	Remove terms smaller than minTermSize
```

Comment:
It's written `minTermSize` instead of `maxTermSize` in the docstring part. I saw 2 of them and one `minimum size` instead of `maximum size`.

Reply:
Thanks, corrected it.